### ISSUE TITLE
Midterm Report Peer Review

### ISSUE THREAD

**Issue by aaparashar** [yashmundra13/ORIE4741#8]
This project attempts to predict water contamination of public sources of water given what they know about the fracking site. Using data from EPA and Frac Focus, they removed missing or corrupted data, split it in to a 70% train and 30 test set and code nominal features using one hot encoding. Then they run preliminary linear regressions with least square, lasso and ridge regression models.

One thing that I liked about this report was the detail mentioned in their exploratory data analysis. It was very informative for them to include how they combined the two datasets and I think that the method used to combine the datasets is reasonable. Another thing I liked about this report was analysis of the correlated features. The heat map is a really great way to visually identify strongly correlated features and the fact that they didn't include columns that would obviously be highly correlated was a good choice on their part to ensure the diagram wasn’t misleading. The final aspect I liked was that they tried multiple variations of linear regression to ensure none of the variants could help the prediction before moving on to non linear models in the future.

One thing I didn’t like about the report was that the map was hard to see visually as it was really faint. Also in regards to the map cc0, cc1, …etc. are on the key but they aren’t quantified so it is hard to tell how much lower is cc1 than cc2 and its hard to figure out what the map means so the visual isn’t very helpful to aid in analysis. Another thing I wish they would have done is explain why they don't have a validation set for hyperparameter tuning as well as explicitly stating what evaluation metric they are using and why. i.e. it is stated they are using mean squared error but first the choice is never explained and second other evaluation metrics don't seem to be considered like precision, recall, f1 score or a combination of those. Finally, I wish they would’ve analyzed the results instead of just stating them. The results could’ve been stated in a table (making it much more readable) and then the space could’ve been used to discussing what the results meant and hypothesizing why things happened they way it happened.

I look forward to seeing your final project!
